package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
+  },
+  "peerDependencies": {
+    "react": ">=14.x"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Automattic/gridicons"
+  },
+  "license": "GPL-2.0+",
+  "files": [ "build/index.js", "build/example.js" ],
```

Comment:
FWIW I think this will auto-reformat itself if anyone uses `npm i --save` by collapsing onto multiple lines.